packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -559,7 +559,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             }
             // If the first snapshot request was with blobs and we either timed out or the size was too big, then try to fetch without blobs.
             if ((error.errorType === OdspErrorType.snapshotTooBig || error.errorType === OdspErrorType.fetchTimeout) && snapshotOptions.blobs) {
-                const snapshotOptionsWithoutBlobs: ISnapshotOptions = { ...snapshotOptions, blobs: undefined, mds: undefined };
+                const snapshotOptionsWithoutBlobs: ISnapshotOptions = { ...snapshotOptions, blobs: 0, mds: undefined };
```

Comment:
Can you please also add an assert in fetchSnapshotCore to not allow undefines for properties?

Reply:
I don't think we need an assert. Because in getQueryString we remove all options with undefined. So just did the same in post also.

Reply:
But not for usePost fork, right?

Reply:
I did it inside fetchSnapshotCore.